src/super_gradients/examples/benchmark/benchmark_gif.py

```diff
@@ -0,0 +1,42 @@
+import torch
+from typing import Callable
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+from super_gradients.training.utils.media.video import load_video
+
+
+def benchmark_model(action: Callable, benchmark_cycles: int, warmup_cycles: int):
```

Comment:
I am not sure this is something we wish to show...
It has not been in our favor to show our benchmark results in Python.

Reply:
I took it out